framework/aster-frame/src/arch/x86/timer/mod.rs

```diff
@@ -12,19 +12,20 @@ use spin::Once;
 use trapframe::TrapFrame;
 
 use crate::arch::x86::kernel;
-use crate::config::TIMER_FREQ;
 use crate::sync::SpinLock;
 use crate::trap::IrqLine;
 
 use self::apic::APIC_TIMER_CALLBACK;
 
+/// The timer frequency (Hz)
+pub const TIMER_FREQ: u64 = 1000;
```

Comment:
So why is it preferable to choose this value? Justify the number in the comment.